src/structures/channel.ts

```diff
@@ -49,4 +49,4 @@ export async function createChannel(
   return channel;
 }
 
-export interface Channel extends Unpromise<ReturnType<typeof createChannel>> {}
+export type Channel = Unpromise<ReturnType<typeof createChannel>>;
```

Comment:
Never use `type` for these. Interfaces are so much better. Like no circular import errors with interfaces.

Reply:
I believe a simple deno lint ignore here is better than having this be types